services/grafana-logging/6.60.1/grafana.yaml

```
@@ -39,7 +39,7 @@ data:
   dashboardLink: "/dkp/logging/grafana"
   docsLink: "https://grafana.com/docs/"
   # Check https://artifacthub.io/packages/helm/grafana/grafana/6.58.6 for app version
-  version: "8.5.26"
+  version: "9.5.13"
```

```suggestion
  version: "8.5.26"
```

we need to use an earlier version of grafana for our logging instances https://github.com/mesosphere/kommander-applications/pull/1760